src/seedu/addressbook/data/person/Contact.java

```diff
@@ -0,0 +1,36 @@
+package seedu.addressbook.data.person;
+
+/**
+ * Represents a Person's contact infomation in the address book.
+ * Super class of Address, Phone, Email
+ */
+public class Contact {
+
+    public final String value;
```

Comment:
It's a good practice to limit all class members permission whenever possible.
Consider setting permission to private

Reply:
Personally I think the value can maintain public, because we need to access to the value in AdaptedPerson class, and there is something like phone.value = source.getPhone().value;

Reply:
I have changed it into private, thanks for your advice!